From 8a65ad94c5ca989f74203eb61f6eb3b6d3da2c1e Mon Sep 17 00:00:00 2001
From: Sai Surya Teja <sssteja@habana.ai>
Date: Mon, 22 Nov 2021 13:05:58 +0200
Subject: [PATCH] [SW-0] gaudi2_arm: preboot: size check to 127KB

Modified the preboot size check for gaudi2_arm to 127KB (1KB as a guard)
For preboot size greater than 127KB in gaudi2_arm compilation will
result in build fail

Change-Id: I1f583dab6e7ea26cf358c24f25f6a50afc84bdc3
---

diff --git a/arch/gaudi2/arm/Makefile b/arch/gaudi2/arm/Makefile
index 60c2025..0e276bb 100644
--- a/arch/gaudi2/arm/Makefile
+++ b/arch/gaudi2/arm/Makefile
@@ -2,7 +2,7 @@
 
 FW_SIZE := 0x5400
 PPBOOT_MAX_SIZE := 20480
-PREBOOT_MAX_SIZE := 65536
+PREBOOT_MAX_SIZE := 130048
 
 PREBOOT_SRCS_C += $(shell ls $(PREBOOT_ROOT_DIR)/lib/preboot/thermal_v2.c)
 
